### PR TITLE
1664 p max-width in forms

### PR DESCRIPTION
**Pull request by sutt0n**

### CODE REVIEW

File: web-client/src/styles/overrides.scss [hunk 1]
```diff
@@ -270,4 +270,8 @@ option:first-child {
 
 p {
   max-width: 700px;
+
+  .usa-form & {
```
Comment (kkoskelin) [inline web-client/src/styles/overrides.scss:274]: I confess I do not understand this selector. Can you explain `.usa-form & ` ?

Comment (sutt0n) [reply web-client/src/styles/overrides.scss:274]: It addresses `p` _within_ `usa-form`. For instance:

```html
<div class="usa-form">
 <p>this is the verbiage that will inherit the width</p>
</div>
```

Comment (kkoskelin) [reply web-client/src/styles/overrides.scss:274]: I honestly had no idea that this was possible. Is this a SASS thing?

**Review by kkoskelin — APPROVED** [review]

File: web-client/src/styles/overrides.scss (hunk 1 above)
Comment (sutt0n) [reply web-client/src/styles/overrides.scss:274]: It is a SASS thing: https://css-tricks.com/the-sass-ampersand/#article-header-id-5

**Pull request merged**